nuxt.config.js

```diff
@@ -281,4 +285,8 @@ if (process.env.INTERCOM_APPID) {
   );
 }
 
+if (process.env.NODE_ENV !== 'production') {
+  nuxtConfig.purgeCSS.enabled = true; // force purge in dev
```

Comment:
Seems the `enabled` option could be a function?
https://github.com/Developmint/nuxt-purgecss#enabled

Reply:
I tried, and the param passed into it was always `undefined` when in dev mode